testsuite/features/srv_change_task_schedule.feature

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2015-2019 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Change the schedule of task mgr-sync-refresh-default
+
+  Scenario: Change the schedule of task mgr-sync-refresh-default
+    Given I am authorized as "admin" with password "admin"
+    When I follow "Admin"
+    And I follow "Task Schedules"
+    And I follow "mgr-sync-refresh-default"
+    And I check radio button "weekly"
+    And I select "Friday" from "date_day_week"
+    And I click on "Update Schedule"
+    Then I should see a "Schedule mgr-sync-refresh-default has been updated." text
+    When I follow "Task Schedules"
+    And I follow "mgr-sync-refresh-default"
+    Then I should see a "Friday" text
+    And radio button "weekly" is checked
+    When I check radio button "monthly"
+    And I select "17" from "date_day_month"
+    And I click on "Update Schedule"
+    Then I should see a "Schedule mgr-sync-refresh-default has been updated." text
+    When I follow "Task Schedules"
```

Comment:
start a third scenario before this step

Reply:
No, we are idempotent at feature level, not at scenario level. Sorry I did not make this clear, my bad, I will add a note to the idempotency documentation.

That's the reason why we might have "cleanup" scenarios at the end :smile: .

Please just split into several scenarios, each one with a different title, not changing the steps, for each time you click on the link..

Reply:
Ok. Re-setting up test environment since this needs to be tested before. Stay tuned...